### ISSUE TITLE
Neural network visualisation using html css and js

### ISSUE THREAD

**Issue by muhtalhakhan** [muhtalhakhan/Hacktoberfest2023#41]

**Comment by rituraj12797** [comment]
i would like to work on this issue 
kindly give more info about this one..

**Comment by rituraj12797** [comment]
what is the requirement of this issue

**Comment by muhtalhakhan** [comment]
https://wagenaartje.github.io/neataptic/docs/tutorials/visualization/

**Comment by rituraj12797** [comment]
@muhtalhakhan  most of the work is done i am facing one issue
+

**Comment by rituraj12797** [comment]
why the link distance is elongating overtime instead of extending to a particular limit

**Comment by rituraj12797** [comment]
https://github.com/muhtalhakhan/Hacktoberfest2023/assets/120770641/11673c60-adeb-4ff7-b7e0-b358c38dd171

**Comment by rituraj12797** [comment]
or should i push this only?

**Comment by muhtalhakhan** [comment]
> or should i push this only?
> 
> 

Yeah @rituraj12797 push it and it looks amazing... you can surely discuss this one with me on discord(@muhtalhakhan) to add more features.
